### DISCUSSION TITLE
Add missing beatmaps that are on collection.db

### DISCUSSION THREAD

**Discussion by gabrieldabbah** [ppy/osu#17523] [Ideas] [answered]
For example, you import a collection.db, and Lazer detects which beatmaps you don't have that are included in collection.db and downloads them.

**Comment by peppy** [comment, marked as answer]
`collections.db` is a legacy file. It should generally be used in conjunction with the beatmaps being loaded (they should be next to it).

If you're looking for a way to share collections with others, for the time being the playlists system is a better fit. If you have a different use case, please try explaining what you want to do, rather than a proposed solution.

**Reply by Moltanicaa** [reply]
Same idea - https://github.com/ppy/osu/discussions/13194
